packages/node_modules/@webex/plugin-meetings/src/members/index.js

```diff
@@ -739,6 +739,26 @@ export default class Members extends StatelessWebexPlugin {
     return this.membersRequest.muteMember(options);
   }
 
+  /**
+   * Raise or lower the hand of a member in a meeting
+   * @param {String} memberId
+   * @param {boolean} [raise] - to raise hand (=true) or lower (=false), default: true
+   * @returns {Promise}
+   * @public
+   * @memberof Members
+   */
+  raiseLowerHand(memberId, raise = true) {
```

Comment:
minor: I think the name of the function is not descriptive enough
`raiseOrLowerHand`, `toggleRaiseHand` or `setRaiseHandState`

Reply:
adressed - pls check again